src/api/create.js

```diff
@@ -1,11 +1,30 @@
 import assign from 'object-assign';
+import customElements from '../native/custom-elements';
 import createElement from '../native/create-element';
 import init from './init';
-import registry from '../shared/registry';
+import support from '../native/support';
 
 export default function (name, props) {
-  const Ctor = registry.get(name);
-  const elem = Ctor ? Ctor.type.create(Ctor) : createElement(name);
-  Ctor && init(elem);
+  let elem;
+  const Ctor = customElements.get(name);
```

Comment:
You would have. Each PR was branched off the previous ones since the work is tightly coupled but I still wanted to break them up into chunks. Once we start merging in order we'll see less files in the diffs.